fpa/fastapi/fastapi.go

```diff
@@ -0,0 +1,209 @@
+package fastapi
```

Comment:
The _fastapi_ package is partitioned in three files:
- _fastapi.go_ for general info
- _fastapi_get.go_ for retrieval logic of CIDs
- _fastapi_put.go_ for pinning CIDs.